### ISSUE TITLE
Pcmanfm-qt does not resize wallpaper with lxqt-config-monitor

### ISSUE THREAD

**Issue by ianorlin** [lxde/lxqt#378]
On a  kvm vm running ubuntu 14.10 launch lxqt-config-monitor and have is start at 1024x768 resolution. Then resize the resolution to 1440x900 or another resolution and notice that the wallpaper does not resize and strech again. Although this seems to happen with xrandr as well. 
before 
![desktopbeforereschange](https://cloud.githubusercontent.com/assets/7546640/5042478/a0a3deec-6b8a-11e4-908c-f86f92971c5e.png)
after
![desktopafterresize](https://cloud.githubusercontent.com/assets/7546640/5042534/b86d8996-6b8b-11e4-8f16-eaaf2b785863.jpg)
pcmanfm-qt:

```
  Installed: 0.8.0+bzr269+201410291631~ubuntu14.10.1
  Candidate: 0.8.0+bzr269+201410291631~ubuntu14.10.1
  Version table:
 *** 0.8.0+bzr269+201410291631~ubuntu14.10.1 0
        500 http://ppa.launchpad.net/lubuntu-dev/lubuntu-daily/ubuntu/ utopic/main i386 Packages
        100 /var/lib/dpkg/status
```

lxqt-config:

```
  Installed: 0.8.0+bzr247+201411080047~ubuntu14.10.1
  Candidate: 0.8.0+bzr247+201411080047~ubuntu14.10.1
  Version table:
 *** 0.8.0+bzr247+201411080047~ubuntu14.10.1 0
        500 http://ppa.launchpad.net/lubuntu-dev/lubuntu-daily/ubuntu/ utopic/main i386 Packages
        100 /var/lib/dpkg/status
```

**Comment by PCMan** [comment]
I'm not able to reproduce this bug here. Have you tried the latest version?
Does the bug still exist?
Thanks!

**Comment by pmattern** [comment]
This seems to be some issue specific to QEMU but not to LXQt. At least I could reproduce it in VMs running either LXQt or Xfce lacking the other desktop environment respectively but neither in a VM running KDE nor bare metal independent from the desktop environment.

So what ever it is, I don't think LXQt is to blame here.

**Comment by City-busz** [comment]
I confirm this bug, and it happens only with the background rendered by PCManFM-Qt. Other desktop environments are resize their desktops correctly when I resize the VM.

**Comment by pmattern** [comment]
> Other desktop environments are resize their desktops correctly when I resize the VM.

Confirmed. Unlike in earlier version I can't see it with recent Xfce any longer either.

Problem only appears when the resolution is increased from the session. It can't be seen when the resolution was saved in ~/.config/autostart/lxqt-config-monitor-autostart.desktop by `lxqt-config-desktop` before.
The wallpaper gets displayed at 1024x768 which is the default resolution of a VM when no RandR settings are applied. In earlier versions of Xfce the same resolution seemed to be used for a short time upon launching a session as well as could be deduced from the transient positions of splash "Simple" and panel. So maybe PCManFM-Qt just isn't aware of RandR settings getting applied once the session was launched.
The context menu over that additional black space is always the one of the window manager, whether or not "Show menus provided by windows managers..." in "Desktop Preferences", Tab "Advanced" (`pcmanfm-qt --desktop-pref=advanced`) is checked. This is, unlike the actual desktop, not affected by the problem described in lxde/pcmanfm-qt#197, btw.
Application windows don't suffer from the problem but use the higher resolution that's actually set if maximized.

**Comment by regs01** [comment]
lxqt-config-monitor.conf stored in ~/.config/LXQt folder, while rest of config files stored in ~/.config/lxqt

It just doesn't take into account that *nix has case sensitive file names. Every time you change resolution you have to copy lxqt-config-monitor.conf into ~/.config/lxqt so then it works.

**Comment by pmattern** [comment]
> copy lxqt-config-monitor.conf into ~/.config/lxqt so then it works.

Cannot confirm this running today's VCS checkouts on Arch Linux.
After copying `lxqt-config-monitor.conf` from `~/.config/LXQt` to `~/.config/lxqt` and starting another session the desktop still looks as depicted in the second screenshot of the first comment in KVM/QEMU virtual machines here.

**Comment by pmattern** [comment]
Running latest commits of LXQt including lxde/pcmanfm-qt@c048d72 aka 0.11.3 as well as recent point releases 0.11.x on Debian testing I don't see this problem any longer.
Tests are suggesting this is due to a fix in Qt 5.7.*1* as the problem does get manifest when the very same commits are compiled against Qt 5.7.0. (Unlike I first thought commits lxde/pcmanfm-qt@de24628 and lxde/pcmanfm-qt@84f5d07 don't seem to be involved.)

@ianorlin @City-busz
Any chance you can confirm?

Qt 5.7.1 is available in Arch Linux, Debian testing, Fedora 25 or Rawhide and openSUSE Tumbleweed right now while it's still missing in openSUSE Leap 42.2. Personally I'd tend to say we can just close this as downstream without taking any further action...
